DashAI/front/src/types/plugin.ts

```diff
@@ -0,0 +1,19 @@
+import type { PluginStatus } from "../pages/plugins/constants/pluginStatus";
+
+interface Tag {
```

Comment:
The naming convention was missing! It should be called ITag.

```suggestion
interface ITag {
```